NOTICE

```diff
@@ -2,4 +2,17 @@ Apache Bahir
 Copyright (c) 2016 The Apache Software Foundation.
 
 This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
\ No newline at end of file
+The Apache Software Foundation (http://www.apache.org/).
+
+
+===================================================================
```

Comment:
Git history would show me as the contributor, not Shixiong, but the  [spark-packages](https://github.com/spark-packages) project has been deleted so I will revert the change to the NOTICE file